index-fr.html

```diff
@@ -30,7 +30,7 @@
   <meta property="og:type" content="article" />
   <meta property="og:url" content="https://aatishb.com/maskmath/" />
   <meta property="og:image" content="https://aatishb.com/maskmath/assets/twitter-banner.png" />
-  <meta property="og:description" content="Même des masques partiellement efficaces peuvent stopper la diffusion de la COVID-19, à condition qu'assez de gens en portent." />
+  <meta property="og:description" content="Même des masques partiellement efficaces peuvent stopper la diffusion de COVID-19, à condition qu'assez de gens en portent." />
```

Comment:
I see that you removed "la" before "COVID-19" everywhere. There are [some debates](https://fr.wikipedia.org/wiki/Maladie_%C3%A0_coronavirus_2019#Europe_francophone) on whether COVID-19 should be considered masculine or feminine (i.e. used with "le" or "la"). In general, official wording uses the feminine gender, but popular usage seem to be masculine.

In any case, removing "la" entirely doesn't seem correct to me. We could either have "la diffusion de la COVID-19" or "la diffusion du COVID-19" (since "de le" is conflated into "du" in French). But "la diffusion de COVID-19" doesn't work well, it gives the impression that COVID-19 is some kind of brand or private noun.